### PR TITLE
Add New-GcLogMetric cmdlet

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/441)

### CODE REVIEW

File: Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1 [hunk 1]
```diff
@@ -61,3 +61,168 @@ Describe "Get-GcLogMetric" {
...
     }
 }
+
+Describe "New-GcLogMetric" {
+    It "should work with -LogName" {
+        $r = Get-Random
+        $metricName = "gcps-new-gclogmetric-$r"
+        $logName = "gcps-new-gclogmetric-log-$r"
+        try {
+            New-GcLogMetric $metricName -LogName $logName
```
Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1]: Why are you not setting $createdMetric here? New-GcLogMetric does return the metric.

File: Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1 [hunk 2]
```diff
@@ -61,3 +61,168 @@ Describe "Get-GcLogMetric" {
         { Get-GcLogMetric -Metric "non-existent-metric-name" -ErrorAction Stop } | Should Throw "does not exist"
     }
 }
+
+Describe "New-GcLogMetric" {
```
Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1]: Should you add a test that New-GcLogMetric and Get-GcLogMetric return the same value?

**Comment by quoctruong** [comment]
@ILMTitan PTAL

**Comment by ILMTitan** [comment]
I did not mean you had to check every time. Once would have been sufficient. Extra work aside, LGTM.

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**